### DISCUSSION TITLE
Patch new line add

### DISCUSSION THREAD

**Discussion by mberbero** [coder/code-server#2207] [General] [answered]
Hi everyone
I'm actually now try Liveshare. And i take error. Error message says liveshare package needs extensionAllowedProposedApi . So i want to add line to pacth file but i didnt know that. The line I want to add is below.

Thank you for your help.

`"ms-vsliveshare.vsliveshare",`

![image](https://user-images.githubusercontent.com/40326463/96268337-9d812480-0fd1-11eb-9f1d-55c2b812a1fa.png)

**Comment by code-asher** [comment, marked as answer]
You can enable the proposed API with a flag when starting code-server
(requires version 3.5.0 of code-server or later).

```
code-server --enable-proposed-api <extension id>
```

Keep in mind though that LiveShare is closed source and only licensed to
work with VS Code. I don't think it works on code-server. See
https://github.com/cdr/code-server/issues/388.

**Reply by mberbero** [reply]
I actually ran liveshare. I can open Liveshare broadcasts but cannot join streams. Other than that, voice call does not work. Chat and other functions are working. If you are interested, you can contact

**Reply by code-asher** [reply]
Neat! In that case `--enable-proposed-api` might be all you need.

**Reply by desnudopenguino** [reply]
> I actually ran liveshare. I can open Liveshare broadcasts but cannot join streams. Other than that, voice call does not work. Chat and other functions are working. If you are interested, you can contact

I'm trying to get this working myself, I got code-server v3.8.0 installed on an alpine linux vm i'm using to build a project. and I installed the extension through the UI, so the icon shows up on the left side nav, but nothing happens when I click share or join. do you have the steps you used, or something that might be able to help me get this running?

**Reply by mberbero** [reply]
I think you need to install the dependencies, so there is a script file.

Link: https://gist.githubusercontent.com/mberbero/3056d8900cae1c27ff85e53a36559263/raw/62597d63fdeafac82a33097d00a10f09c04446ae/liveshare.sh
